src/components/standalone/users_objects/CreateOrEditHostSetDrawer.vue

```diff
@@ -133,6 +145,36 @@ function runFieldValidators(
   return validators.every((validator) => validator.valid)
 }
 
+async function getMatchedItemsName(matches: string[]): Promise<string> {
+  try {
+    const res = await ubusCall('ns.objects', 'get-info', { ids: matches })
+    const names = []
+    for (const match of Object.values(res.data.info) as any[]) {
```

Comment:
Risky call, a Type would be ideal to handle this kind of data
While it's only a type-safe oriented fix (and if API changes at runtime this will fail) let's strengthen the codebase

Reply:
Should be good @Tbaile, is it what you got in mind ?

Reply:
An example of type definitions and all can be found here: https://github.com/NethServer/nethsecurity-ui/blob/f018a2a4d73267899fbd310287b5610aa8c759b1/src/components/standalone/security/ips/IpsEventList.vue#L55-L70